app/api/api_v1/schemas/document.py

```diff
@@ -153,6 +154,7 @@ class DocumentUpdateRequest(BaseModel):
     md5_sum: Optional[str] = None
     content_type: Optional[str] = None
     cdn_object: Optional[str] = None
+    document_status: Optional[DocumentStatus] = None
```

Comment:
I can see why you are leaving this in - but I don't know if this could be just enough rope to hang ourselves with :)

Reply:
Actually that was an oversight, I will remove it